extras/Cargo.toml

```diff
@@ -8,11 +8,12 @@ repository = "https://github.com/Zondax/filecoin-signing-tools"
 description ="Temporary lib used for compatibility with wasm"
 
 [dependencies]
-forest_vm = { git = "https://github.com/zondax/forest", branch="lola/txnid-params-multisig"  }
-forest_address = { git = "https://github.com/zondax/forest", branch="lola/txnid-params-multisig"  }
-forest_encoding = { git = "https://github.com/zondax/forest", branch="lola/txnid-params-multisig"  }
-forest_cid = { git = "https://github.com/zondax/forest", branch="lola/txnid-params-multisig"  }
-num_bigint = { package = "forest_bigint" , git = "https://github.com/zondax/forest", branch="lola/txnid-params-multisig"  }
-clock = { git = "https://github.com/zondax/forest", branch="lola/txnid-params-multisig"  }
+forest_vm = "0.2.0"
+forest_message = "0.3.0"
+forest_address = "0.2.0"
+forest_encoding = "0.1.1"
+forest_cid = "0.1.0"
+num_bigint = { package = "forest_bigint", version = "0.1.0"}
+clock = { git = "https://github.com/chainsafe/forest", branch="master" }
```

Comment:
This seems to be a problem... it is failing in CI
Let's check with forest if they have release a crate

Reply:
They haven't released this crate I checked yesterday.
And this is what is needed : https://chainsafe.github.io/forest/clock/struct.ChainEpochClock.html

Reply:
I have find the error they don't have a `master` banch anymore only a `main`.